esp/esp/program/models/__init__.py

```diff
@@ -1067,18 +1067,28 @@ def getShirtInfo(self):
         if 'class_approved' in teacher_dict:
             query = teacher_dict['class_approved']
             query = query.filter(registrationprofile__most_recent_profile=True)
-            query = query.values_list('registrationprofile__teacher_info__shirt_type',
+            if Tag.getTag('teacherinfo_shirt_type_selection') == 'False':
+                query = query.values_list('registrationprofile__teacher_info__shirt_size')
+                query = query.annotate(people=Count('id', distinct=True))
+
+                for row in query:
+                    shirt_size, count = row
+                    shirt_count['M'][shirt_size] = count
+
+            else:
+                query = query.values_list('registrationprofile__teacher_info__shirt_type',
                                       'registrationprofile__teacher_info__shirt_size')
```

Comment:
minor nit: can you fix the indentation of this line?  It should line up to the string right above.  I'm not sure why the linter isn't complaining here!